### PR TITLE
Added storage account changes in make.json for PublishTestResultsV2

### PR DESCRIPTION
**Pull request by virekumar**
**Task name**: PublishTestResultsV2

**Description**: Changed location of external resources present in make.json 

**Documentation changes required:**  Not required

**Added unit tests:**  Not required

**Attached related issue:** No

**Checklist**:
- [X] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [X] Checked that applied changes work as expected

**Testing**
**Test tab pre change**
image

**Test tab post change**
image

### CODE REVIEW

File: Tasks/PublishTestResultsV2/make.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "externals": {
     "archivePackages": [
       {
-        "url": "https://testmanagementstore.blob.core.windows.net/testmanagementptrcontainer/29161277/PublishTestResults.zip",
+        "url": "https://testmanagementstore.z13.web.core.windows.net/testmanagementptrcontainer/29161277/PublishTestResults.zip",
```
Comment (adityashahms) [inline Tasks/PublishTestResultsV2/make.json:5]: What is this URL?

Comment (adityashahms) [inline Tasks/PublishTestResultsV2/make.json:5]: We might have to update the signing pipeline first, to ensure that we are able to publish a latest version of PTR library in this new location first.

File: Tasks/PublishTestResultsV2/task.json [hunk 2]
```diff
@@ -1,158 +1,158 @@
...
-    "visibility": [
-        "Build",
-        "Release"
-    ],
-    "author": "Microsoft Corporation",
-    "version": {
-        "Major": 2,
-        "Minor": 247,
-        "Patch": 0
+  "id": "0B0F01ED-7DDE-43FF-9CBB-E48954DAF9B1",
```
Comment (adityashahms) [inline Tasks/PublishTestResultsV2/task.json:2]: The only change in this file should be on the line 16(something similar to task.loc.json). Can you verify once through file diff, that we are not missing anything while updating this file?

File: Tasks/PublishTestResultsV2/make.json (hunk 1 above)
Comment (virekumar) [reply Tasks/PublishTestResultsV2/make.json:5]: We are using azure web static container from same storage account.

[Static website hosting in Azure Storage | Microsoft Learn](https://learn.microsoft.com/en-us/azure/storage/blobs/storage-blob-static-website)

File: Tasks/PublishTestResultsV2/task.json (hunk 2 above)
Comment (virekumar) [reply Tasks/PublishTestResultsV2/task.json:2]: verified. No changes at rest places

File: Tasks/PublishTestResultsV2/make.json (hunk 1 above)
Comment (virekumar) [reply Tasks/PublishTestResultsV2/make.json:5]: Updated Release as well

Comment (virekumar) [reply Tasks/PublishTestResultsV2/make.json:5]: https://dev.azure.com/mseng/AzureDevOps/_releaseProgress?releaseId=21585947&_a=release-task-editor&environmentId=212631715

**Review by eswarnathp — APPROVED** [review]
lgtm

**Review by adityashahms — APPROVED** [review]

**Pull request merged**